addons/crm_claim/views/crm_claim_menu.xml

```diff
@@ -47,11 +45,9 @@
         </record>
 
 
-        <menuitem name="Claims" id="menu_crm_case_claims"
+        <menuitem id="menu_crm_case_claims" name="Claims"
```

Comment:
In this file every diff is Unusual, If i am wrong please correct me
![giphy2](https://cloud.githubusercontent.com/assets/14940179/12153334/a75ed418-b476-11e5-9768-1c512825a460.gif)